test/cli.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import spy from "sinon/lib/sinon/spy.js";
+import { expect } from "chai";
+import * as path from "path";
+import { loadUserFunctionFromDirectory } from "../src/user-function.js";
+import cli, { parseArgs } from "../src/cli.js";
+
+let workers = 0;
+const fakeThrong = async function (...processes: Array<any>) {
+  workers = processes[0].count;
+  return await processes[0].worker("worker 1", function () {
+    return null;
+  });
+};
+
+let args = [
+  "/some/path/to/node",
+  "/some/path/to/cli-binary.js",
+  "serve",
+  "./fixtures/js-esm-template",
+];
+
+describe("cli.ts", async () => {
```

Comment:
Do we need tests for any of these?
- `WEB_CONCURRENCY`
- `WEB_CONCURRENCY` and `-w` (with the latter taking priority)

Reply:
Yes! I'll add them =)